### ISSUE TITLE
recap installer vs makefile

### ISSUE THREAD

**Issue by tonyskapunk** [rackerlabs/recap#90]
Seems a duplicated effort having two scripts that produce almost the same outcome, here the differences between them:

1. `Recap-installer` is more explicit with directory permissions than `Makefile`:
  ```log
  | Directory                | recap-installer | Makefile |
| /var/log/recap           |            0700 |     0750 |
| /var/log/recap/backups   |            0700 |     0750 |
| /var/log/recap/snapshots |            0700 |     0750 |
```
2. The `recap-installer` places the docs in a [directory dependent of the version](https://github.com/rackerlabs/recap/blob/master/util/recap-installer#L54-L56), The [makefile is generic](https://github.com/rackerlabs/recap/blob/master/Makefile#L30-L31).
1. The `recap-installer` does not include `recaptool.8` man page
1. The `recap-installer` [gzips the man pages](https://github.com/rackerlabs/recap/blob/master/util/recap-installer#L61-L63), while the `Makefile` does not, but for instance the [RPM spec takes care of this](https://github.com/rackerlabs/recap/blob/master/util/packaging/rpm/recap.spec#L47-L49).
1. The `recap-installer` provides support to [macosx for the cronjob](https://github.com/rackerlabs/recap/blob/master/util/recap-installer#L61-L63), `Makefile` does not.
1. The `recap-installer` looks for the [httpd config directory](https://github.com/rackerlabs/recap/blob/master/util/recap-installer#L76-L81) if found the [recap.httpd.conf](https://github.com/rackerlabs/recap/blob/master/src/recap.httpd.conf) file is copied over, this is disabled by default, i.e commented out.
1. `Makefile` is more flexible than `recap-installer` on the path to be installed.

**Comment by carlwgeorge** [comment]
IIRC the original intent was for the Makefile to obsolete the installer script.  Some extra notes:

* Regarding 2, check out https://fedoraproject.org/wiki/Changes/UnversionedDocdirs.  That only applies to Fedora, and RHEL/EPEL packages still use the old versioned style.  It is handled automatically by the `%doc` and `%license` macros, so we don't have to adjust for it in the spec file.  But that makes me realize that from a packaging perspective we don't want the docs installed by `make install`, as we handle them with those macros.  I'll open a separate issue to discuss creating a standalone `doc` target in the Makefile.
* Regarding 4, check out https://fedoraproject.org/wiki/Packaging:Guidelines#Manpages for the Fedora guideline on this.

**Comment by tonyskapunk** [comment]
@carlwgeorge thanks for that info, that pretty much translates to advantages on the `Makefile` for (2) keep using `$(DESTDIR)$(DOCDIR)/recap`  and for (4) don't compressing man pages.

> When installing man pages, note that they should be installed uncompressed as the build system will compress them as needed.

**Comment by tonyskapunk** [comment]
Additionally, there is no way to `uninstall` using the script, another good reason to get rid of it :)

**Comment by tonyskapunk** [comment]
Fix on #89
